pylint/checkers/base.py

```diff
@@ -1719,7 +1719,31 @@ def _create_naming_options():
     return tuple(name_options)
 
 
-class NameChecker(_BasicChecker):
+class NameCheckerHelper:
+    """Class containing functions required by NameChecker and NonAsciiNamesChecker"""
+
+    def _check_name(
+        self, node_type: str, name: str, node: nodes.NodeNG, optional_kwarg: Any = None
+    ):
+        """Only Dummy function will be overwritten by implementing classes
+
+        Note: kwarg arguments will be different in implementing classes
+        """
+        raise NotImplementedError
+
+    def _recursive_check_names(self, args: Iterable[nodes.AssignName]):
+        """check names in a possibly recursive list <arg>"""
+        for arg in args:
+            if isinstance(arg, nodes.AssignName):
+                self._check_name("argument", arg.name, arg)
+            else:
+                # pylint: disable-next=fixme
+                # TODO: Check if we can remove this if branch because of
```

Comment:
Which version are you referring to?

Reply:
I don't know. I haven't written the code, I just moved it. But it seems to be not up to date anymore, that why I put the TODO in it.
Never change a running system, that you not responsible for and don't understand ;-).

Reply:
Ah makes sense! I'll add this to my personal to-do list for a follow-up 😄